scripts/remove_after_use/divorce_quickfiles.py

```diff
@@ -0,0 +1,178 @@
+from osf.models import QuickFolder, OSFUser, UserLog
+from osf.models.legacy_quickfiles import QuickFilesNode
+from django.core.paginator import Paginator
+from django.db.models import OuterRef, Subquery, Count
+from django.contrib.contenttypes.models import ContentType
+
+import logging
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def transfer_logs(quickfiles_node):
+    """
+    Recasts logs and saves them, old logs will be deleted with all quickfilesnodes.
+    :param quickfiles_node: a QuickFilesNode
+    :return:
+    """
+    for log in quickfiles_node.logs.all():
+        log.__class__ = UserLog
+        log.save()
+
+
+def reverse_transfer_logs(quickfiles_node):
+    """
+    Recasts logs and saves them, old logs will be deleted with all quickfilesnodes.
+    :param quickfiles_node: a QuickFilesNode
+    :return:
+    """
+    for log in quickfiles_node.logs.all():
+        log.__class__ = NodeLog
+        log.save()
+
+
+def create_quickfolders():
+    """
+    Bulk creates a Quickfolder for every user.
+    :return:
+    """
+    users = OSFUser.objects.all()
+    user_content_type_id = ContentType.objects.get_for_model(OSFUser).id
+    quickfolder_content_type_id = ContentType.objects.get_for_model(QuickFolder).id
+
+    paginated_users = Paginator(users, 1000)
+    logger.info('There are {} '.format(users.count()))
+
+    total_created = 0
+    quickfolders_to_create = []
+    for page_num in paginated_users.page_range:
+        for user in paginated_users.page(page_num).object_list:
+            quickfolder = QuickFolder(target_object_id=user.id,
+                                      target_content_type_id=user_content_type_id,
+                                      provider='osfstorage',
+                                      path='/')
+
+            quickfolders_to_create.append(quickfolder)
+
+            total_created += 1
+    QuickFolder.objects.bulk_create(quickfolders_to_create)
```

Comment:
`bulk_create` has a [batch size](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-create) parameter - this is too many to do at once.